challenge-03/README.md

```diff
@@ -150,8 +179,32 @@ deve conter no retorno da frase acima é "metro" no lugar de "metros".
 método), que será concatenada com a frase de retorno, mostrando a resposta
 correta, de acordo com os dados inseridos no objeto.
 */
-?
+pessoa.apresentacao = function() {
+	var retorno;
+	if(pessoa.sexo === 'Masculino') {
+		retorno = 'Olá, eu sou o ' + pessoa.nome + ' ' + pessoa.sobrenome + ', ';
+	}else{
+		retorno = 'Olá, eu sou a ' + pessoa.nome + ' ' + pessoa.sobrenome + ', ';
+	}
+
+	if(pessoa.idade === 1) {
+		retorno += 'tenho ' + pessoa.idade + ' ano, ';
+	}else{
+		retorno += 'tenho ' + pessoa.idade + ' anos, ';
+	}
+
+	retorno += pessoa.altura + 'm, ';
+	retorno += 'meu peso é ' + pessoa.peso + 'Kg ';
+
+	if(pessoa.caminhouQuantosMetros === 1) {
+		retorno +=  'e, só hoje, eu já caminhei ' + pessoa.caminhouQuantosMetros  + ' metro!';
+	}else{
+		retorno +=  'e, só hoje, eu já caminhei ' + pessoa.caminhouQuantosMetros  + ' metros!';
```

Comment:
Só uma dica: não é necessário repetir a frase toda duas vezes, pois se você tiver que fazer alguma alteração, terá que mudar em dois lugares diferentes. O ideal é você modificar somente as partes que precisam ser modificadas.

Nesse caso, você nem precisaria do `else`. Poderia declarar as variáveis no início da função com os valores padrão, e usar o `if`  somente para modificar o valor, quando necessário :)